book/src/faq.md

```diff
@@ -101,6 +88,27 @@ involve converting the methods in question to take `&self` argument instead of `
 
 Please consult [`MessageStream`] documentation for details.
 
+## Why aren't property values updating for my service that doesn't notify changes?
+
+A common issue might arise when using a zbus proxy is that your proxy's property values aren't 
+updating. This is due to zbus' default caching policy, which updates the value of a property only
+when a change is signaled, primarily to minimize latency and optimize client request performance.
+By default, if your service does not emit change notifications, the property values will not
+update accordingly. 
+
+However, you can explicitly bypass this caching mechanism for a client:
+### 1. Client built via `dbus_proxy`
+By adding the `#[dbus_proxy(property(emits_changed_signal = "false"))]` annotation to your proxy
+
+### 2. Client built via `ProxyBuilder`
```

Comment:
* Some vertical spacing please.
* bypassing -> disabling caching for specific properties.
* A few other rewordings (see below).

```suggestion
However, you can disabling caching for specific properties:

### 1. `dbus_proxy`-generated proxies

By adding the `#[dbus_proxy(property(emits_changed_signal = "false"))]` annotation to your property.

### 2. `Proxy` instances

```

Reply:
Oh and I think it's best to also list disabling of property caching for all the properties of a proxy through `ProxyBuilder::cache_properites` (which works for both kinds of proxies.